imports/plugins/core/ui/client/components/checkbox/checkbox.js

```diff
@@ -3,6 +3,16 @@ import PropTypes from "prop-types";
 import { Components, registerComponent } from "@reactioncommerce/reaction-components";
 
 class Checkbox extends Component {
+  // Create a semi-random ID
+  //
+  // ID's are required for form controls and their labels for ARIA compliance.
+  // This will be the default ID for this component unless once is passed in through props.
```

Comment:
Can you clarify, comment is a little confusing

Reply:
When you have a form control, like an input `<input />` and you have a label for that input `<label />`. The label must have a prop `htmlFor` `<label htmlFor="idOfInputField>My Label</label>`. This then requires your input to have an ID `<input id="idOfInputField" />`. 

Thus, an ID prop is required for all of our form components, as they include labels by default. And the getter in question ensures there's at least a default.